src/Version.h

```diff
@@ -0,0 +1,38 @@
+//
+//  version.h
+//  drafter
+//
+//  Created by Jiri Kratochvil on 2015-02-13
+//  Attribution Notice:
+//  This work might use parts of Node.js `node_version.h`
+//  https://github.com/joyent/node
+//
+//  Copyright (c) 2014 Apiary Inc. All rights reserved.
+//
+
+#ifndef DRAFTER_VERSION_H
+#define DRAFTER_VERSION_H
+
+#define DRAFTER_MAJOR_VERSION 0
+#define DRAFTER_MINOR_VERSION 1
+#define DRAFTER_PATCH_VERSION 0
+
+#define DRAFTER_VERSION_IS_RELEASE 0
+
+#ifndef DRAFTER_STRINGIFY
```

Comment:
hmm.. I am not sure, but we don't need this `ifndef`. Right? `ifndef DRAFTER_VERSION_H` would take care of this.